internal/compiler-interface/src/main/datatype/other.json

```diff
@@ -16,7 +16,7 @@
 				},
 				{
 					"name": "api",
-					"type": "Companions"
+					"type": "xsbti.Maybe<Companions>"
```

Comment:
When I save to `inc_compile` I throw this away, and it thaws back as `Maybe.nothing()` value here, which should help in both time and space.